modules/openapi-generator/src/main/resources/kotlin-client/build.gradle.mustache

```diff
@@ -26,13 +26,17 @@ buildscript {
     }
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-serialization:$kotlin_version"
```

Comment:
@kuFEAR I think this line should only be applied when using kotlin serialization, or not?

Reply:
right, thanks I fixed it

Reply:
Thanks